### PR TITLE
fix: swapper slippage popover trigger

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Fixes a regression introduced by yours truly in https://github.com/shapeshift/web/pull/6584 - see https://github.com/chakra-ui/chakra-ui/issues/2843 for reference on why this doesn't work.

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

> What protocols, transaction types or contract interactions might be affected by this PR?

Low, this *is* a fix

## Testing

- Slippage gear icon click is happy again for swapper
- Slippage gear icon hover is still showing a disabled tooltip for LP

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

### This diff

Screenshot 2024-03-28 at 13 46 15
Screenshot 2024-03-28 at 13 46 28

### Develop - happy LP but dead click on swapper

https://github.com/shapeshift/web/assets/17035424/6deac485-e9c2-47fd-9cc2-24c19c9792a5

image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6601** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6601?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/MultiHopTrade/components/SlippagePopover.tsx [hunk 1]
```diff
@@ -122,16 +123,18 @@ export const SlippagePopover: FC<SlippagePopoverProps> = memo(
...
-            <IconButton
-              aria-label={translate('trade.tradeSettings')}
-              icon={faGear}
-              variant='ghost'
-              isDisabled={isDisabled}
-            />
-          </Tooltip>
-        </PopoverTrigger>
+        <Tooltip isDisabled={!isDisabled} label={translate(tooltipTranslation)}>
+          <Box display='inline-block'>
```
Comment (gomesalexandre) [inline src/components/MultiHopTrade/components/SlippagePopover.tsx:127]: The magic bit - this simply wraps `<PopoverTrigger />` in a `<Box />`, and moves the `<Tooltip />` wrapping outside of the box/tooltip node.

View without whitespaces diff for the actual diff

image

**Review by kaladinlight — APPROVED** [review]
Working as expected now. Is there a reason we don't just hide that on the LP view instead of disable?

**Comment by gomesalexandre** [comment]
> Working as expected now. Is there a reason we don't just hide that on the LP view instead of disable?

There is ! This was the initial way I've gone with this but the flex layout becomes all kinds of borked:

image

Adding spacers didn't produce so nice alignment either.

I did discuss this with @reallybeard and we agreed on disabling as a simpler solution.

**Comment by kaladinlight** [comment]
> > Working as expected now. Is there a reason we don't just hide that on the LP view instead of disable?
> 
> There is ! This was the initial way I've gone with this but the flex layout becomes all kinds of borked:
> 
> image
> 
> Adding spacers didn't produce so nice alignment either.
> 
> I did discuss this with @reallybeard and we agreed on disabling as a simpler solution.

Roger that. Freaking web dev :P

**Pull request merged**